packages/drawer/src/test/mwc-drawer.test.ts

```diff
@@ -16,12 +16,32 @@
  */
 
 import {Drawer} from '@material/mwc-drawer';
+import {html} from 'lit-html';
+
+import {fixture, TestFixture} from '../../../../test/src/util/helpers';
 
 const SCRIM_SELECTOR = '.mdc-drawer-scrim';
 const HEADER_SELECTOR = '.mdc-drawer__header';
 const DISMISSIBLE_CLASS = 'mdc-drawer--dismissible';
 
+interface DrawerProps {
+  hasHeader: boolean;
+  type: string;
+}
+
+const defaultDrawer = html`<mwc-drawer></mwc-drawer>`;
+
+const drawer = (propsInit: Partial<DrawerProps>) => {
+  return html`
+    <mwc-drawer
+      ?hasHeader=${propsInit.hasHeader === true}
+      .type=${propsInit.type ?? ''}>
```

Comment:
TBH I'm surprised this works in this version of TS

Reply:
3.7 apparently is what we have and thats when it was introduced, good times :D